### ISSUE TITLE
Using SignalR with edge

### ISSUE THREAD

**Issue by kalanbates** [tjanczuk/edge#206]
TL;DR;
How can I leverage edge.js to allow my Express server to expose SignalR hubs?
(Edit: after thinking about this a bit more -I'd just started my investigation- if I'm using Owin.SelfHosting to expose my hubs, there shouldn't really be anything for edge or Express to worry about. I should just attach a client to the /signalR resources exposed by my host application)

---

Lets say -as a hypothetical- that I have an Owin self-hosted server that exposes a SignalR hub.  And let's say that I have an existing Express server.  

I want to use SignalR behind my server wall for my WebSockets implementation so that I can leverage the power of .NET.  I'm not terribly concerned with what technology the clients must use; the overriding factor would be ease of implementation.  So, if it's easiest to expose Express.io capabilities to clients and route to a SignalR hub, that would be great.  If it's easiest to use Express as a passthrough to SignalR from end to end, that would be great too.

I'm currently only using edge to expose CLR functions to Express api routes (which is ridiculously simple, by the way; thank you for that) but not really sure where to begin with integrating SignalR.

Is this something that is feasible?
